app/dataMappers/dataBase.js

```diff
@@ -0,0 +1,7 @@
+const { Client } = require('pg');
+
+const client = new Client();
```

Comment:
J'avoue avoir une préférence pour l'approche explicite : 
```suggestion
const client = new Client(process.env.PG_URL);
```

que l'approche implicite avec les variable `PG_HOST`, etc... mais à toi de voir :)